portfolio/src/main/webapp/script.js

```diff
@@ -75,7 +75,13 @@ function getKdramaRecommendation() {
   const recommendation = recommendations[Math.floor(Math.random() * recommendations.length)];
 
   // Add it to the page.
-  document.getElementById("kdrama-cover").src = recommendation.cover;
+  document.getElementById("kdrama-cover").src       = recommendation.cover;
   document.getElementById("kdrama-title").innerText = recommendation.title;
-  document.getElementById("kdrama-title").href = recommendation.link;
+  document.getElementById("kdrama-title").href      = recommendation.link;
```

Comment:
I know it looks good to align the RHS but it's usually one space character before/after the `=` :)